hazelcast/src/test/java/com/hazelcast/nio/IOUtilTest.java

```diff
@@ -807,81 +858,120 @@ public void testGetPath_whenPathsInvalid() {
         getPath();
     }
 
-    @Test
-    public void testCopy_shouldDeepCopy() throws IOException {
-        File target = createFile("destination");
-        File source = createFile("source");
-        writeTo(source, "test content");
+    private File newFile(String filename) {
+        File file = new File(getFilename(filename));
+        files.add(file);
+        return file;
+    }
+
+    private File newFile(File parent, String filename) {
+        File file = new File(parent, getFilename(filename));
+        files.add(file);
+        return file;
+    }
+
+    private String getFilename(String filename) {
+        String name = "IOUtilTest-" + testName.getMethodName() + "-" + filename;
```

Comment:
This adds the method name to created files, so the tests are now independent, even if the same short filenames like `source` or `target` are used in the tests. This fixed some follow-up errors on Windows, when one of the tests failed. This also allows to run all test methods in parallel now.